rebalance/rebalance.py

```diff
@@ -128,6 +128,49 @@ def calc_optimal_amount(out_ours, out_total, in_ours, in_total, payload):
     raise RpcError("rebalance", payload, {'message': 'rebalancing these channels will make things worse'})
 
 
+class NoRouteException(Exception):
+    pass
+
+
+def getroute_basic(plugin: Plugin, targetid, fromid, excludes, msatoshi: Millisatoshi):
+    try:
+        return plugin.rpc.getroute(targetid,
+                                   fromid=fromid,
+                                   exclude=excludes,
+                                   msatoshi=msatoshi,
```

Comment:
The `getroute_basic` method currently ignores the `plugin.msatfactor_effective`.
I know that's likely by intention, just want to point that our at line 534, it will be set to 1 for `rebalanceall`.

Not sure how you see that, maybe we want to set the `getroute_xyz` method as a parameter to the `rebalance` method. I can see the point in using basic for `rebalanceall`.

Reply:
It can be a parameter to the `rebalance` method of course. On the other hand, it usually makes the code cleaner, if we do not use too many parameters for functions. And `rebalance` already has 8 parameters.

Reply:
Yes I know. But if we add this as a parameter, we can remove the two "maxhops" and "msatfactor" as they were only added in order to have `rebalanceall` setting other values. And we are down to 7 parameters :)